ui/src/app/components/messages.tsx

```diff
@@ -0,0 +1,77 @@
+import {FlexRow} from 'app/components/flex';
+import {ClrIcon} from 'app/components/icons';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import * as React from 'react';
+
+const MessageWithIcon = ({
+  messageType = 'warning' || 'error' || 'success',
```

Comment:
This doesn't look like it's working as intended since the || strings are no-ops, i.e. this expression will always evaluate to `'warning'`. If you're trying to declare a type, you can instead add a type clause:

```
{
  messageType,
  iconPosition,
  iconSize = 30,
  children
} : {
  messageType: 'warning'|'error'|'success',
  iconPosition: 'top'|'center'|'bottom',
  iconSize: number,
  children
}
```